lib/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Thu Nov 19 02:40:40 IST 2020
+sdk.dir=C\:\\Users\\Nikhil\\AppData\\Local\\Android\\Sdk
```

Comment:
Again same as above. 
"Please remove this from source control. This is a local file."